app/assets/stylesheets/components/_filter-summary.scss

```diff
@@ -0,0 +1,60 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-summary__remove-filters {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.app-c-filter-summary__remove-filter {
+  border: 1px solid $govuk-border-colour;
+  border-radius: govuk-em(5, 16px);
+  padding: govuk-spacing(2);
+  margin: 0 govuk-spacing(2) govuk-spacing(1) 0;
+  text-decoration: none;
+  display: inline-block;
+  color: $govuk-text-colour;
+  background-color: govuk-colour("light-grey");
+  @include govuk-font(16);
+
+  @include govuk-media-query($from: tablet) {
+    text-wrap: nowrap;
```

Comment:
Don't think we need this, no? It's unlikely a label will be wide enough on tablet+ to wrap, but if it needs to, it should: 
<img width="1086" alt="image" src="https://github.com/user-attachments/assets/ddd6297b-1b72-45b5-97ca-ebd1c15e4db5">

Reply:
no that shouldnt be there anymore